task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +524,38 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let result;
+
+    // Array of successful combinations
+    let successCombinArr = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  
+    // Loop to change voids to null and then apply the flat method
+    for(let i = 0; i < position.length; i++) {
+      position[i].length = 3;
+  
+      for(let j = 0; j < position[i].length; j++) {
+        if(position[i][j] !== 'X' && position[i][j] !== '0') {
+          position[i][j] = null;
```

Comment:
Do not change function arguments if not required. 
you don't need to replace undefined with null. just use successArr.includes(undefiend).

Reply:
I need to change arguments with null, because input on some cases look like : (3) [Array(3), Array(2), Array(3)]
0: (3) ["X", "X", "X"]
1: (2) ["0", "0"]
2: (3) [empty × 2, "X"]

Reply:
array with index 1 has two arguments, but it should have 3 arguments. And  method flat() remove this void places